### PR TITLE
Add themes sim web

### PR DESCRIPTION
**Pull request by yauyy**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md]
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

### Description
sim-web

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/sim-web.omp.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "right",
+      "filler": " ",
```
Comment (JanDeDobbeleer) [inline themes/sim-web.omp.json]: why is there an empty filler?

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
One question, please have a look.

File: themes/sim-web.omp.json (hunk 1 above)
Comment (yauyy) [reply themes/sim-web.omp.json]: Sorry, did not run the correct command to generate the image, resulting in a style exception

File: themes/sim-web.omp.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "right",
+      "filler": "",
```
Comment (JanDeDobbeleer) [inline themes/sim-web.omp.json]: this is the default and can be removed ;-)

File: themes/sim-web.omp.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      "newline": true,
```
Comment (JanDeDobbeleer) [inline themes/sim-web.omp.json]: is there a specific reason to start with a newline?

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
I would do the following, move the second block to the first position and make the first block a right aligned one so the prompt spans 2 lines and not three.

File: themes/sim-web.omp.json (hunk 2 above)
Comment (yauyy) [reply themes/sim-web.omp.json]: ok~

File: themes/sim-web.omp.json (hunk 3 above)
Comment (yauyy) [reply themes/sim-web.omp.json]: Wanted a little distance from the last command~

**Comment by yauyy** [comment]
> 我将执行以下操作，将第二个块移动到第一个位置并使第一个块右对齐，以便提示跨越 2 行而不是 3 行。

Want it to be unaffected by the path

**Comment by yauyy** [comment]
> I would do the following, move the second block to the first position and make the first block a right aligned one so the prompt spans 2 lines and not three.

Want it to be unaffected by the path

**Comment by bend-n** [comment]
I think it needs to be in the `themes` folder, not `主题`

**Comment by JanDeDobbeleer** [comment]
@bend-n ooh, good catch. I missed that.

**Review by bend-n — CHANGES_REQUESTED** [review]
Needs to be in the `themes` folder.

**Comment by yauyy** [comment]
> Needs to be in the `themes` folder.

Thanks for the reminder that a plugin causes

**Review by yauyy — COMMENTED** [review]
update

**Pull request merged**